src/java.base/share/classes/jdk/internal/module/ModuleHashes.java

```diff
@@ -138,6 +138,7 @@ private static byte[] computeHash(ModuleReader reader, String algorithm) {
      * Computes a hash from the names and content of a module.
      *
      * @param supplier supplies the module reader to access the module content
+     *
```

Comment:
Revert, there are no other changes to ModuleHashes.java

Reply:
Fixed.